doc/install/build-ceph.rst

```diff
@@ -61,15 +61,14 @@ Advanced Package Tool (APT)
 
 To create ``.deb`` packages for Debian/Ubuntu, ensure that you have cloned the 
 `Ceph`_ repository, installed the `Build Prerequisites`_ and installed 
-``debhelper``::
+``debhelper`` and ``dpkg``::
 
-	sudo apt-get install debhelper
+	sudo apt-get install debhelper dpkg
 
-Once you have installed debhelper, you can build the packages::
+Once you have installed these packages, you can build the Ceph packages using
+the ``make-debs.sh`` build script included in the distribution::
 
-	sudo dpkg-buildpackage
-
-For multi-processor CPUs use the ``-j`` option to accelerate the build.
+	sudo ./make-debs.sh
```

Comment:
how about using `dpkg-buildpackage -rfakeroot` in `make-debs.sh` to avoid running this script as a superuser?

why do you want to use `make-debs.sh` instead of using `dpkg-buildpackage` directly? the former also sets up the a debian repo using `reprepro`. i don't think that's in the scope of building debian packages.

Reply:
@tchaikov thanks for the comments. I made these doc changes based on the conversation in http://tracker.ceph.com/issues/39486 - it probably makes sense to clarify these questions there? For the sake of getting the dashboard docs changes finally merged, I'll take out the changes in `build-ceph.rst` for now, until these points have been clarified.

Reply:
quote

> update: build new debian packages using "make-debs.sh" which is included in source. these packages include all frontend assets and thus i could enable dasboard. so the documentation should be updated.

@LenzGr i see. actually i don't understand why `make-deb.sh` works, while `dpkg-buildpackage` does not ..

Reply:
I've removed the commit that changes the build instructions now, the changes to `dashboard.rst` have not been touched.